lib/galaxy/model/__init__.py

```diff
@@ -10085,7 +10085,7 @@ class Page(Base, HasTags, Dictifiable, RepresentById, UsesCreateAndUpdateTime):
     )
     title = Column(TEXT)
     deleted = Column(Boolean, index=True, default=False)
-    importable = Column(Boolean, index=True, default=False)
+    importable = Column(Boolean, index=True, default=False, key="_importable")
```

Comment:
I'm curious, what does this do? :thinking: 

To be honest, I couldn't understand very well the documentation I found for it:

> An optional string identifier which will identify this ``Column`` object on the :class:`_schema.Table`. When a key is provided, this is the only identifier referencing the ``Column`` within the application, including ORM attribute mapping; the ``name`` field is used only when rendering SQL.

Reply:
Good catch, this was when I thought maybe i'd write a decorator so that page.importable would provide False for null values!